examples/TESSLightCurves.ipynb

```diff
@@ -0,0 +1,275 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "463d13f3-c28b-4bc4-89cf-332ceeb55663",
+   "metadata": {},
+   "source": [
+    "## <u> Demonstrate some generic techniques for sonifying 1D data:</u>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "37f21355",
+   "metadata": {},
+   "source": [
+    "**First, import relevant modules:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f64e736a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%reload_ext autoreload \n",
+    "%autoreload 2\n",
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "from strauss.sonification import Sonification\n",
+    "from strauss.sources import Objects\n",
+    "from strauss import channels\n",
+    "from strauss.score import Score\n",
+    "from strauss.generator import Synthesizer\n",
+    "import IPython.display as ipd\n",
+    "import os\n",
+    "from scipy.interpolate import interp1d\n",
+    "import numpy as np\n",
+    "import requests\n",
+    "import json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a3e9c2-e9e3-48ca-a21c-0ae64bb549f6",
+   "metadata": {},
+   "source": [
+    "**Now, we load some test data!**\n",
+    "\n",
+    "We use simulated subjects from the PH-TESS dataset:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbf7b1b5-79b7-4a2a-a8e9-15f196a15ca6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "s94812564 = 'https://panoptes-uploads.zooniverse.org/subject_location/c4a2a54a-da3b-4f1d-9823-702549a2806a.txt'\n",
+    "s94880899 = 'https://panoptes-uploads.zooniverse.org/subject_location/03caefe6-7d48-4225-9cc0-e5a60f337af8.txt'\n",
+    "s94880882 = 'https://panoptes-uploads.zooniverse.org/subject_location/7578bd5d-b8da-4c42-8da7-5f897ded018e.txt'\n",
+    "response = json.loads(requests.get(s94812564).text)\n",
+    "days = np.array(response['x'])\n",
+    "flux = np.array(response['y'])\n",
+    "normalised_days = lambda d: d / days.max()\n",
+    "x = np.array(list(map(normalised_days, days)))\n",
+    "normalised_flux = lambda f: ((f - flux.min()) / (flux.max() - flux.min()))\n",
+    "y = np.array(list(map(normalised_flux, flux)))\n",
+    "\n",
+    "\n",
+    "plt.plot(days,y)\n",
+    "plt.ylabel('Normalised Flux')\n",
+    "plt.xlabel('Days')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0f6edf6-59bf-4bdc-a9e7-c7f097cd08d3",
+   "metadata": {},
+   "source": [
+    "**Set up some universal sonification parameters and classes for the examples below**\n",
+    "\n",
+    "For all examples we use the `Synthesizer` generator to create a 30 second, mono sonification."
```

Comment:
I think this line is a holdover from an earlier notebook. Anyway, the examples here are 20 seconds and stereo (panning from left to right as we move along the light curve.)